api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -5784,7 +5784,7 @@
             "description": "HTTPGet specifies the http request to perform."
           },
           "initialDelaySeconds": {
-            "description": "Number of seconds after the container has started before liveness probes are initiated. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes",
+            "description": "Number of seconds after the container has started before startup, liveness or readiness probes are initiated. More info: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle#container-probes",
```

Comment:
The openapi file is automatically generated and you should not modify it directly. Instead, change the position of the following field and then regenerate it using `./hack/update-openapi-spec.sh`
https://github.com/kubernetes/kubernetes/blob/dc6d8c6b582dea09b355363af9612d1439dad09a/staging/src/k8s.io/api/core/v1/types.go#L2283

Reply:
Hi @mengjiao-liu , Im sorry, accidentally deleted this PR. 
After changing the types and successfully running `./hack/update-openapi-spec.sh` I dont see the changes being applied to `api/openapi-spec/swagger.json` or any other file that this text appears. Is there any other command that I should run prior to that ?
I tried following this docs that seems outdated https://kubernetes.io/docs/contribute/generate-ref-docs/contribute-upstream/#making-changes-to-the-upstream-source-code

Reply:
> Hi @mengjiao-liu , Im sorry, accidentally deleted this PR.
After changing the types and successfully running ./hack/update-openapi-spec.sh I dont see the changes being applied to api/openapi-spec/swagger.json or any other file that this text appears. Is there any other command that I should run prior to that ?

sorry, I missed some coaand.
you can run following command:
```
./hack/update-codegen.sh
./hack/update-openapi-spec.sh
```
or 
```
# may need to run for a long time 
make update 
```
> I tried following this docs that seems outdated https://kubernetes.io/docs/contribute/generate-ref-docs/contribute-upstream/#making-changes-to-the-upstream-source-code

loos like the docs is outdated.

Reply:
Thanks @mengjiao-liu , I ran both commands and now I can see swagger.json being updated. I updated issue #119596 with a separate commit containing just the files touched by both commands. What sounds off to me is that it is also updating some files that I haven't changed the spec, such as flowcontrol, storage, batch...  Is that expected ?